src/app/pages/HomePage/index.tsx

```diff
@@ -111,34 +124,50 @@ export const HomePage: FC = () => {
   const showInfoScreenBtn = isMobile && step !== ParaTimeSelectorStep.ShowHelpScreen
 
   return (
-    <HomepageLayout>
-      <Content>
-        <LogotypeBox>
-          <Logotype showText />
-        </LogotypeBox>
-        <SearchInputContainer>
-          <SearchInputBox>
-            <Search disabled={isApiOffline} variant={searchVariant} onFocusChange={onFocusChange} />
-          </SearchInputBox>
-          {isApiOffline && (
-            <Box sx={{ display: 'flex', justifyContent: 'center' }}>
-              <OfflineIndicator />
-            </Box>
-          )}
-        </SearchInputContainer>
-        <Box sx={{ zIndex: 1 }}>
-          <ParaTimeSelector step={step} setStep={setStep} disabled={searchHasFocus} />
-        </Box>
-      </Content>
+    <>
+      <HomepageLayout>
+        <Content>
+          <LogotypeBox>
+            <Logotype showText />
+          </LogotypeBox>
+          <SearchInputContainer>
+            <SearchInputBox>
+              <Search disabled={isApiOffline} variant={searchVariant} onFocusChange={onFocusChange} />
+            </SearchInputBox>
+            {isApiOffline && (
+              <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+                <OfflineIndicator />
+              </Box>
+            )}
+          </SearchInputContainer>
+          <Box sx={{ zIndex: zIndexHomePage.paraTimeSelector }}>
+            <ParaTimeSelector step={step} setStep={setStep} disabled={searchHasFocus} />
+          </Box>
+        </Content>
 
-      <FooterStyled>
-        {showInfoScreenBtn && (
-          <IconButton aria-label={infoAriaLabel} onClick={onToggleInfoScreenClick}>
-            <InfoOutlinedIcon fontSize="medium" sx={{ color: 'white' }} />
-          </IconButton>
-        )}
-        {!isMobile && <Footer />}
-      </FooterStyled>
-    </HomepageLayout>
+        <FooterStyled>
+          {showInfoScreenBtn && (
+            <IconButton aria-label={infoAriaLabel} onClick={onToggleInfoScreenClick}>
+              <InfoOutlinedIcon fontSize="medium" sx={{ color: 'white' }} />
+            </IconButton>
+          )}
+          {!isMobile && <Footer />}
+        </FooterStyled>
+      </HomepageLayout>
+      {activeMobileGraphTooltip && (
+        <GraphTooltipMobile
```

Comment:
Why it's not in the `ParaTimeSelector`?

Reply:
Due to `position: relative` in the `Graph` components.

Reply:
Resolved.